conf/application.conf

```diff
@@ -44,6 +44,7 @@ play.http.requestHandler = "uk.gov.hmrc.play.bootstrap.http.RequestHandler"
 
```

Comment:
I would suggest double check the microservice module we use from bootstrap library. Looks like the current `uk.gov.hmrc.play.bootstrap.MicroserviceModule` is deprecated in favour of a new `uk.gov.hmrc.play.bootstrap.backend.BackendModule`

Reply:
Updated